forge-gui/res/cardsfolder/upcoming/grisly_sigil.txt

```diff
@@ -0,0 +1,10 @@
+Name:Grisly Sigil
+ManaCost:B
+Types:Sorcery
+K:Casualty:1
+A:SP$ DealDamage | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Choose target creature or planeswalker | NumDmg$ Y | SubAbility$ DBGainLife | SpellDescription$ Choose target creature or planeswalker. If it was dealt noncombat damage this turn, Grisly Sigil deals 3 damage to it and you gain 3 life. Otherwise, Grisly Sigil deals 1 damage to it and you gain 1 life.
```

Comment:
This will always give you 3 Life because it damages first...

Might have to Branch it?

Reply:
💯- ha ha, I had gotten to that point in testing while out of town, set it aside, and then forgot about this issue - thanks for reminding me 😬 

I should have put a comment in the script when I caught that issue